types/jsrsasign/modules/KJUR/asn1/cms/SignedData.d.ts

```diff
@@ -31,7 +31,7 @@ declare namespace jsrsasign.KJUR.asn1.cms {
      */
     class SignedData extends ASN1Object {
         signerInfoList: SignerInfo[];
-        constructor();
+        constructor(params?: object);
```

Comment:
`object` is suspicious; this is usually not a very helpful type or accurate.

The PR description asks for a link to back up the changes in the PR. Can you point to some sort of source for this parameter?

Reply:
https://kjur.github.io/jsrsasign/api/symbols/KJUR.asn1.cms.SignedData.html

Reply:
The method accepts JSON objects with a flexible schema, allowing any structure or format

Reply:
I see, okay.